Query/FilterCondition.php

```diff
@@ -29,6 +29,7 @@ class FilterCondition
     const METHOD_NIN = "NIN";
     const METHOD_BOOLEAN = "BOOLEAN";
     const METHOD_IS_NULL = "IS_NULL";
+    const METHOD_IS_NNULL = "IS_NOT_NULL";
```

Comment:
This probably needs a better naming, especially the "NN" part (I just used the "NIN" naming scheme)